### PR TITLE
Mas 422 l3

### PR DESCRIPTION
**Pull request by markreesmoj**
Work in progress
It is tricky to run two different versions of the application

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -101,8 +101,13 @@ tasks {
       .singleFile
     jvmArgs = listOf("-javaagent:$jacocoAgent=destfile=$buildDir/jacoco/cucumber.exec,append=false")
   }
+  val versionTwo by registering(Test::class) {
```
Comment (nickmcmahon01) [inline build.gradle.kts:104]: This is getting messy, lets sort this out Thursday.

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/ProtectLevelCalculator.kt [hunk 2]
```diff
@@ -72,19 +74,21 @@ class ProtectLevelCalculator(
...
 
   private fun getMappaPoints(mappa: Mappa?): Int =
     when (mappa) {
-      M3, M2 -> 30
+      M3, M2 -> levelA()
       M1 -> 5
       else -> 0
     }
 
+  private fun levelA() = if (calculationVersion == 2) 300 else 30
```
Comment (nickmcmahon01) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/ProtectLevelCalculator.kt:90]: I might be picking on the name, but the points and the level are different things.
What's the 300 for? it is a magic number?

Comment (markreesmoj) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/ProtectLevelCalculator.kt:90]: yes, 30 and 300 are magic numbers. We will see what happens today but this is a way of keeping the link between points scored and protect tier

File: build.gradle.kts (hunk 1 above)
Comment (markreesmoj) [reply build.gradle.kts:104]: It's a nightmare. We also need to consider the BDD tests which are an accident waiting to happen. We have got three versions of the application now and the tests cannot keep up as they are currently structured

**Comment by markreesmoj** [comment]
Merging so we can see what it does in preproduction

**Pull request merged**